jina/drivers/queryset/__init__.py

```diff
@@ -0,0 +1,42 @@
+from typing import List
+
+from .. import BaseDriver
+
+
+class QuerySetDriver(BaseDriver):
+    """
+    A QuerySet represents a collection of objects from your database.
+    It can have zero, one or many filters. Filters narrow down the query results
+    based on the given parameters.
+
+    In SQL terms, a QuerySet equates to a SELECT statement,
+    and a filter is a limiting clause such as WHERE or LIMIT.
+    """
+
+    def __init__(self, queryset_raw: List[str] = None, combine_policy: str = 'append', *args, **kwargs):
```

Comment:
Should not queryset_raw be of type Union[str, List[str]] if we are considering both cases in the body?